packages/devtools-connect/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  root: true,
+  extends: [
+    'semistandard',
+    'plugin:promise/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  rules: {
```

Comment:
is fine for a follow up, but generally we try to avoid doing this, and instead we align on eslint / ts when we move things here, that way we can keep maintaining everything in the same way, especially for stuff that can be just addressed with `eslint --fix`

Reply:
yeah, this will have to be a follow up 🙈 . I mention this in the description - I have tried to make this work with the same rulesets as the other packages, but I kept seeing errors that I couldn't even figure out where they were coming from. so in the end I gave up and used the old eslint config + a couple of exceptions to support the new prettier config.